core/contracts/Voting.sol

```diff
@@ -66,6 +68,16 @@ contract Voting is Testable, MultiRole, OracleInterface {
     // The set of identifiers the oracle can provide verified prices for.
     mapping(bytes32 => bool) private supportedIdentifiers;
 
+    // Percentage of the total token supply that must be used in a vote to create a valid price resolution.
+    // 1 == 100%.
+    FixedPoint.Unsigned private gatPercentage;
+
+    // Reference to the voting token.
+    VotingToken private votingToken;
+
+    // Voting rounds -> voting token snapshot IDs.
+    mapping(uint => uint) private snapshotIds;
```

Comment:
Could this be part of the `Round` struct?

Reply:
Done.